.gitignore

```diff
@@ -73,4 +73,24 @@ conjur_git_commit
 # AuthnOIDC V2 w/ Identity setup
 dev/policies/authenticators/authn-oidc/identity-users.yml
 
+gem/slosilo/*.gem
+gem/slosilo/*.rbc
+gem/slosilo/.bundle
+gem/slosilo/.yardoc
+gem/slosilo/InstalledFiles
+gem/slosilo/_yardoc
+gem/slosilo/coverage
+gem/slosilo/doc/
+gem/slosilo/lib/bundler/man
+gem/slosilo/pkg
+gem/slosilo/rdoc
+gem/slosilo/spec/reports
+gem/slosilo/test/tmp
+gem/slosilo/test/version_tmp
+gem/slosilo/tmp
+gem/slosilo/.rvmrc
+gem/slosilo/.project
+gem/slosilo/.kateproject.d
+gem/slosilo/.idea
+
```

Comment:
Why are we ignoring all the Slosilo files?

Reply:
these are the ignored files frm .gitignore file the was in slosilo library, 
these are not all the slosilo files..